data/json/mapgen/nested/nested_chunks_roof.json

```
@@ -308,24 +308,23 @@
       "rotation": [ 0, 3 ],
       "rows": [
         "%#++#%",
-        "%p__p%",
-        "#p__p#",
-        "#p__p#",
-        "%p__p%",
+        "%1__1%",
+        "#1__1#",
+        "#1__1#",
+        "%1__1%",
         "%#++#%"
       ],
       "terrain": {
         "_": "t_null",
-        "p": "t_grass_long",
         "#": "t_reinforced_glass_shutter_open",
         "+": "t_reinforced_door_glass_c",
         "%": "t_reinforced_glass_shutter",
         "$": "t_wall_glass"
       },
       "furniture": {
```

This got single-lined because it was shortened.
```suggestion
      "furniture": { "S": "f_statue" },
```